packages/core/src/pagination/pagination.stories.ts

```diff
@@ -0,0 +1,312 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@cds/core/button/register.js';
+import { getElementStorybookArgTypes } from '@cds/core/internal';
+import '@cds/core/pagination/register.js';
+import { html } from 'lit-html';
+import customElements from '../../dist/core/custom-elements.json';
+
+export default {
+  title: 'Stories/Pagination',
+  component: 'cds-pagination',
+  argTypes: getElementStorybookArgTypes('cds-pagination', customElements),
+  parameters: {
+    options: { showPanel: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/v2mkhzKQdhECXOx8BElgdA/Clarity-UI-Library---light-2.2.0?node-id=51%3A673',
+    },
+  },
+};
+// pager, pagination, paginator
+export const API = (args: any) => {
```

Comment:
Do we need `args` here?

Reply:
No. I was meaning to remove but it slipped through. Thanks for catching it.